packages/plugin-metrics/test/unit/spec/metrics.js

```diff
@@ -54,12 +86,12 @@ describe(`plugin-metrics`, () => {
     });
 
     describe(`#submitClientMetrics()`, () => {
-      it(`submits a metric to clientmetrics`, () => {
+      it(`submits a metric to clientmetrics if after auth`, () => {
```

Comment:
add a second set of describe nesting here: describe(`before login`), describe(`after login`)